### PR TITLE
Gutenberg/improvement

### PR DESCRIPTION
**Pull request by du-phan**
* Add code-env
* Add support for py3
* Update BeautifulSoup syntax

https://app.clubhouse.io/dataiku/story/44055/gutenberg-support-for-py3
https://app.clubhouse.io/dataiku/story/44056/gutenberbg-update-syntax-for-beautifulsoup

### CODE REVIEW

**Comment by JoachimZ** [comment]
Hello world works in Python2 and Python3?

**Comment by du-phan** [comment]
I tested with python2 and 3 (it's on dev04).
Tmr I'll look at the basicConfig stuff!

File: gutenberg/python-connectors/gutenberg-project-book/connector.py [hunk 1]
```diff
@@ -36,7 +47,6 @@ def generate_rows(self, dataset_schema=None, dataset_partitioning=None,
         lid = len(str(self.book_id))
```
Comment (laurenmoos) [inline gutenberg/python-connectors/gutenberg-project-book/connector.py]: having a simple example of what this looks like as a comment would be helpful

File: gutenberg/python-connectors/gutenberg-project-book/connector.py [hunk 2]
```diff
@@ -36,7 +47,6 @@ def generate_rows(self, dataset_schema=None, dataset_partitioning=None,
...
         fullbid = str(self.book_id)
         rootbid = fullbid # sometimes the id to access a file has a variation, ex fullbid=14285-8 for the book 14285
-        print type(lid)
 
         stopit = 0
```
Comment (laurenmoos) [inline gutenberg/python-connectors/gutenberg-project-book/connector.py]: maybe use a boolean?

**Pull request merged**